xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -1097,6 +1143,9 @@ int CGUIDialogVideoInfo::ManageVideoItem(const CFileItemPtr &item)
 
     // set or change movie set the movie belongs to
     buttons.Add(CONTEXT_BUTTON_SET_MOVIESET, 20465);
+
+    // set movie version
+    buttons.Add(CONTEXT_BUTTON_SET_MOVIE_VERSION, 39301);
```

Comment:
Does this comment have any real value as it basically only repeats what `CONTEXT_BUTTON_SET_MOVIE_VERSION` already implies?

Reply:
Just keep the same code style as the code before.